src/InputToACObject.cs

```diff
@@ -0,0 +1,336 @@
+﻿using Sprache;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+
+namespace ACConfigBuilder
+{
+    public class InputToACObject
+    {
+        public void getIdentNameAndValue(string line, out bool configureExit, out bool subIdentExit, out string subIdent, out string subIdentValue) // parses the Indentifyer for the current block of the configuration
+        {
+            subIdent = ParserGrammar.getsubident.Parse(line);
+            configureExit = false;
+            subIdentExit = false;
+            subIdentValue = String.Empty;
+            if (subIdent == ParserVariables.networkDev ||
```

Comment:
Change the if to a switch to make it more readable

```suggestion
            swith(subIdent){
               case ParserVariables.networkDev:
               case ParserVariables.interfaceNetwokIf :
               case ParserVariables.proxySet :
               case ParserVariables.proxyIp :
               case ParserVariables.exit:     
                  if (subIdent == ParserVariables.exit)
                  {
                     configureExit = true;
                      subIdentExit = true;
                     return;
                 }
                 subIdentExit = false;
                 subIdentValue = ParserGrammar.subidentvalue.Parse(line);
                 break;
            }
```